app/controllers/chat_channels_controller.rb

```diff
@@ -31,7 +31,11 @@ def create
 
   def update
     ChatChannelUpdateService.new(@chat_channel, chat_channel_params).update
-    flash[:settings_notice] = "Channel Settings Updated."
+    if @chat_channel.valid?
```

Comment:
I think it makes more sense to rely on the result of `update` call instead of `@chat_channel.valid?` . In some cases the channel could be valid but still not updated for other reasons.

Reply:
what error to show in case `@chat_channel` is valid and still not updated?
`"Channel Settings Updation Failed. Try Latter"` will work or something else?

Reply:
I would go with smth like "Channel Settings were not updated".